### PR TITLE
Add Simplic.Forms documentation.

### PR DESCRIPTION
**Pull request by MaxPatyk**

### CODE REVIEW

File: support/Misc/Forms/forms.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+The Form-Designer serves as a construction kit for the customer to create his own forms completely according to his ideas.
+This way the customer will be able to modify his own forms without the help of SIMPLIC development and both parties will save time and the customer will be even more flexible. For example, QM forms can be created to record quality characteristics. 
+## Generate new Form
+First the User click "Konfiguration" and then Form Designer to start the Form Designer.
+![Taskbar](~/images/Forms/taskbar.jpg)
+Otherwise the User can search and then clicking Form Designer to start it.
+![Search](~/images/Forms/searchbar.jpg)
+The Grid is open and with a "right Click" or press the Key "F8" a new Form is open.
+![NewFormButton](~/images/Forms/newFormButton.jpg)
+When a Form exists the User can edit it with a "right click" or press the Key "F5" and the Form will open.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:12]:
```suggestion
When a form exists the user can edit it with a "right click" or press the Key "F5" and the form will open.
```

File: support/Misc/Forms/forms.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ ![EditFormButton](~/images/Forms/editFormButton.jpg)
+ The Guid will be created automatic when the Form is open, We will need the Guid later.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:14]:
```suggestion
 The Guid will be created automatically when the form is opened, we will need the Guid later.
```

File: support/Misc/Forms/forms.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+ The new Form need a "Name", "Beschreibung" and a "Tabellen Name".
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:15]:
```suggestion
 The new form need a "Name", "Beschreibung" and a "Tabellen Name".
```

File: support/Misc/Forms/forms.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+ After that we need a "Datenstruktur" with a "right click" in the discolored area, the User can create a new data.
+![NewTreeviewItem](~/images/Forms/newTreeviewItem.jpg)
+A new Form is open with "Feldname" and "Datentyp".
+The "Feldname" is the name of the attribut.
+The "Datentyp" is the datatype of the attribut.
+With clicking "Speichern und schließen" the Form will be closed and saved.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:21]:
```suggestion
With clicking "Speichern und schließen" the form will be closed and saved. 
```

File: support/Misc/Forms/forms.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+ After that we need a "Datenstruktur" with a "right click" in the discolored area, the User can create a new data.
+![NewTreeviewItem](~/images/Forms/newTreeviewItem.jpg)
+A new Form is open with "Feldname" and "Datentyp".
+The "Feldname" is the name of the attribut.
+The "Datentyp" is the datatype of the attribut.
+With clicking "Speichern und schließen" the Form will be closed and saved. 
+![NewTreeviewItemForm](~/images/Forms/newTreeviewItemForm.jpg)
+When the "Datenstruktur" complete is, the User can switch to Design and create the actual Form.
+![SwitchDesign](~/images/Forms/switchDesign.jpg)
+Now the User see a new Form with three columns.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:25]:
```suggestion
Now the User sees a new form with three columns.
```

File: support/Misc/Forms/forms.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+Left is the ToolBox.
+In the middle is the Designer Surface.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:27]:
```suggestion
In the middle is the designer surface.
```

File: support/Misc/Forms/forms.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+![NewTreeviewItem](~/images/Forms/newTreeviewItem.jpg)
+A new Form is open with "Feldname" and "Datentyp".
+The "Feldname" is the name of the attribut.
+The "Datentyp" is the datatype of the attribut.
+With clicking "Speichern und schließen" the Form will be closed and saved. 
+![NewTreeviewItemForm](~/images/Forms/newTreeviewItemForm.jpg)
+When the "Datenstruktur" complete is, the User can switch to Design and create the actual Form.
+![SwitchDesign](~/images/Forms/switchDesign.jpg)
+Now the User see a new Form with three columns.
+Left is the ToolBox.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:26]:
```suggestion
Left is the tool box.
```

File: support/Misc/Forms/forms.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+In the middle is the Designer Surface.
+Right is the Attribute List.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:28]:
```suggestion
Right is the attribute list.
```

File: support/Misc/Forms/forms.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+![DesignTab](~/images/Forms/designTab.jpg)
+First the User choose a item from the ToolBox make a "left click"  and drag into the Designer Surface in the middle.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:30]:
```suggestion
First, the user selects an item from the tool box and drags it to the designer surface in the center
```

File: support/Misc/Forms/forms.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+![SelectDesignerItem](~/images/Forms/selectDesignerItem.jpg)
+![ItemDesignSurface](~/images/Forms/itemDesignSurface.jpg)
+The User choose a item in the Design Surface then in the right the Attribute List will be open.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:33]:
```suggestion
The user selects an item from the design surface and gets the attribute list by right clicking on it.
```

File: support/Misc/Forms/forms.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+Now the User see a new Form with three columns.
+Left is the ToolBox.
+In the middle is the Designer Surface.
+Right is the Attribute List.
+![DesignTab](~/images/Forms/designTab.jpg)
+First the User choose a item from the ToolBox make a "left click"  and drag into the Designer Surface in the middle.
+![SelectDesignerItem](~/images/Forms/selectDesignerItem.jpg)
+![ItemDesignSurface](~/images/Forms/itemDesignSurface.jpg)
+The User choose a item in the Design Surface then in the right the Attribute List will be open. 
+For example the item is a TextBox so the User do a "right click" in "Text" and choose "Binding".
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:34]:
```suggestion
For example: The item is a TextBox and the user right-clicks in "Text" and selects "Binding".
```

File: support/Misc/Forms/forms.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+![SelectBindingText](~/images/Forms/selectBindingText.jpg)
+In the Attribute List the attribut "Text" will be expanded.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:36]:
```suggestion
In the attribute list the attribut "Text" will be expanded.
```

File: support/Misc/Forms/forms.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+Three attributes are very Important.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:37]:
```suggestion
Three attributes are very important.
```

File: support/Misc/Forms/forms.md [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+**Mode** it must be TwoWay
+**Path** the name of the data from the "Datenstruktur"
+**UpdateSourceTrigger** it must be PropertyChanged
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:40]: Some more description why PropertyChanged should be the default option would be nice. In some cases the user might not want it that way

File: support/Misc/Forms/forms.md [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+![ImportantBinding](~/images/Forms/importantBinding.jpg)
+Under the Designer Surface are the 2 tabs Xaml and Designer. By default you are in the Designer, but if you want to see the added tools and settings as Xaml,
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:42]:
```suggestion
Below the designer surface there are two tabs Xaml and Designer. By default you are in the Designer tab, but if you want to see the added tools and settings as Xaml,  
```

File: support/Misc/Forms/forms.md [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+In the Attribute List the attribut "Text" will be expanded.
+Three attributes are very Important.
+**Mode** it must be TwoWay
+**Path** the name of the data from the "Datenstruktur"
+**UpdateSourceTrigger** it must be PropertyChanged
+![ImportantBinding](~/images/Forms/importantBinding.jpg)
+Under the Designer Surface are the 2 tabs Xaml and Designer. By default you are in the Designer, but if you want to see the added tools and settings as Xaml, 
+you can easily navigate there by clicking on Xaml.
+![ClickXaml](~/images/Forms/clickXaml.jpg)
+Everytime the User can testing the Form with clicking "Ausführen" and see the Form in a Window where it will be also later.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:45]:
```suggestion
The user can test the form at any time by clicking "Ausführen" and will see the form in a seperate window.
```

File: support/Misc/Forms/forms.md [hunk 17]
```diff
@@ -0,0 +1,63 @@
...
+![TestingForm](~/images/Forms/testingForm.jpg)
+When the User click "Speichern und schließen" in the Main Window then the Form will be saved.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:47]:
```suggestion
When the user clicks "Speichern und schließen" in the main window, the form is saved.
```

File: support/Misc/Forms/forms.md [hunk 18]
```diff
@@ -0,0 +1,63 @@
...
+## Create Database Table
+The User must create the DataBase Table.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:49]:
```suggestion
The user must create the database table.
```

File: support/Misc/Forms/forms.md [hunk 19]
```diff
@@ -0,0 +1,63 @@
...
+The name of the Table must be the same from the Designer name.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:50]:
```suggestion
The name of the table must be the same as the designer name.
```

File: support/Misc/Forms/forms.md [hunk 20]
```diff
@@ -0,0 +1,63 @@
...
+The Columns "Guid", "TenantId", "IsDeleted", "CreateDateTime", "UpdateDateTime", "CreateUserId", "UpdateUserId" and "ConfigurationGuid" are always important.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:51]:
```suggestion
The columns "Guid", "TenantId", "IsDeleted", "CreateDateTime", "UpdateDateTime", "CreateUserId", "UpdateUserId" and "ConfigurationGuid" are always important.
```

File: support/Misc/Forms/forms.md [hunk 21]
```diff
@@ -0,0 +1,63 @@
...
+The other Columns must have the same name and datatyp from the Form Designer.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:52]:
```suggestion
The other columns must have the same name and data type as in the form designer.
```

File: support/Misc/Forms/forms.md [hunk 22]
```diff
@@ -0,0 +1,63 @@
...
+![DataBase](~/images/Forms/dataBase.jpg)
+## Create Grid
+When the Database is created then the User must create a Grid.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:55]:
```suggestion
Once the database table is created, the user must create a grid.
```

File: support/Misc/Forms/forms.md [hunk 23]
```diff
@@ -0,0 +1,63 @@
...
+Important in Grid "Menü" are **Klasse**, **Namespace**, **Statische Methode**.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:56]:
```suggestion
Important in grid "Menü" are **Klasse**, **Namespace**, **Statische Methode**.
```

File: support/Misc/Forms/forms.md [hunk 24]
```diff
@@ -0,0 +1,63 @@
...
+The three attributes are in the following picture.
+**Name-Parameter Name** and **Wert-Parameter Name** are important too.
+The Parameter "Name" is always "ConfigurationId" and the "Wert" is the Guid from the Form.
```
Comment (MEichhoff) [inline support/Misc/Forms/forms.md:59]:
```suggestion
The parameter "Name" is always "ConfigurationId" and the "Wert" is the guid from the Form.
```

**Comment by simplicbe** [comment]
@MaxPatyk .png is the better format for the web.

**Pull request merged**